File: GridView/RadRadioButtonCellElement/radradiobuttoncellelementvb-zip/RadRadioButtonCellElementVB/RadRadioButtonCellElementVB/RadioButtonColumn.vb
Imports Telerik.WinControls.UI

Public Class RadioButtonColumn

    Inherits Telerik.WinControls.UI.GridViewDataColumn

    Public Sub New(ByVal fieldName As String)
        MyBase.New(fieldName)
    End Sub

    Public Overrides Function GetCellType(ByVal row As GridViewRowInfo) As System.Type
        If TypeOf row Is GridViewDataRowInfo Then
            Return GetType(RadioButtonCellElement)
        End If
        Return MyBase.GetCellType(row)
    End Function

End Class

File: GridView/RadRadioButtonCellElement/radradiobuttoncellelementcs-zip/RadRadioButtonCellElementCS/RadRadioButtonCellElementCS/RadioButtonColumn.cs
using System;
using System.Collections.Generic;
using System.Text;
using Telerik.WinControls.UI;

namespace RadRadioButtonCellElementCS
{
    public class RadioButtonColumn: GridViewDataColumn
    {
        public RadioButtonColumn(string fieldName)
            : base(fieldName)
        {
        }

        public override Type GetCellType(GridViewRowInfo row)
        {
            if (row is GridViewDataRowInfo)
            {
                return typeof(RadioButtonCellElement);
            }        
            return base.GetCellType(row);
        }
    }
}
